ui/l10n/en.js

```diff
@@ -1816,7 +1816,7 @@ var dictionary = {
 "label.volume":"Volume",
 "label.volume.details":"Volume details",
 "label.volume.empty":"No volumes attached to this VM",
-"label.volume.ids":"Volume ID's",
+"label.volume.ids":"Delete Volumes",
```

Comment:
@dhlaluku I would recommend keeping `label.volume.ids` as it is ("Volume ID's"). Instead, you can use the existing `"label.delete.volumes":"Volumes to be deleted"` when listing the volumes IDs to be deleted.

Reply:
Fixed and reverted back the Volume ID's label

Reply:
Thanks!